src/uraft/main.cc

```diff
@@ -43,7 +43,7 @@ void parseOptions(int argc, char **argv, uRaftController::Options &opt, bool &ma
 	("LOCAL_MASTER_MATOCL_PORT", po::value<int>()->default_value(9421), "local master matocl port")
 	("LOCAL_MASTER_CHECK_PERIOD", po::value<int>()->default_value(250), "local master check status period")
 	("URAFT_ELECTOR_MODE", po::value<int>()->default_value(0), "run in elector mode")
-	("URAFT_GETVERSION_TIMEOUT", po::value<int>()->default_value(50), "getversion timeout (ms)")
+	("URAFT_GETVERSION_TIMEOUT", po::value<int>()->default_value(200), "getversion timeout (ms)")
```

Comment:
Probably because of load_config, see below my comment regarding this.

In any case, if we go ahead and increase this, we need to document it in the config and man files